### PR TITLE
migration to antd v4

### PR DESCRIPTION
**Pull request by xcodebuild**

### CODE REVIEW

**Review by ahonn — APPROVED** [review]
LGTM

File: src/renderer/extensions/whistle/index.tsx [hunk 1]
```diff
@@ -386,28 +388,26 @@ export class WhistleExntension extends Extension {
...
-                                style={{ marginLeft: '5px' }}
-                                className={info.systemProxyClassName}
-                                type={info.systemProxyIcon}
-                            />
-                            <Icon style={{ marginLeft: '10px' }} type="menu" />
-                        </div>
-                    </Dropdown>
-                </>
-            );
+            return <>
```
Comment (richardo2016) [inline src/renderer/extensions/whistle/index.tsx:391]: Maybe it's better to remove this React.Fragment? though it comes for convenience primarily

File: package.json [hunk 2]
```diff
@@ -33,12 +33,14 @@
...
   "dependencies": {
+    "@ant-design/compatible": "^1.0.8",
+    "@ant-design/icons": "^4.6.2",
     "@sentry/electron": "1.1.0",
     "@sentry/node": "5.10.2",
     "@timkendrick/monaco-editor": "^0.0.9",
     "@umijs/hooks": "^1.9.3",
     "@xcodebuild/global-proxy": "^0.0.6",
-    "antd": "^3.26.6",
+    "antd": "^4.16.6",
```
Comment (bairuobing) [inline package.json:43]: Amazing

File: src/renderer/extensions/whistle/index.tsx (hunk 1 above)
Comment (xcodebuild) [reply src/renderer/extensions/whistle/index.tsx]: removed

**Pull request merged**